src/main/java/com/strandls/document/pojo/DocumentMappingList.java

```diff
@@ -0,0 +1,146 @@
+package com.strandls.document.pojo;
+
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.strandls.resource.pojo.UFile;
+import com.strandls.user.pojo.UserIbp;
+//import com.strandls.user.pojo.UserIbp;
+import com.strandls.userGroup.pojo.Featured;
+import com.strandls.userGroup.pojo.UserGroupIbp;
+import com.strandls.utility.pojo.FlagShow;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class DocumentMappingList {
+
+	
+	private Object document;
```

Comment:
unknown object, Please dont use Generic object, Always go it a defined object!!!
please go with a defined object instaed of using Object